packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -19,6 +28,13 @@ import injectSpark from '../../modules/redux-spark/inject-spark';
  */
 export class ChatWidget extends Component {
 
+  constructor(props) {
+    super(props);
+    this.getActivityList = this.getActivityList.bind(this);
+    this.handleScrollToBottom = this.handleScrollToBottom.bind(this);
+    this.handleScroll = _.debounce(this.handleScroll.bind(this), 150);
```

Comment:
Yea. That's how I had it before but the function inside would never fire. Not sure why that was the case and didn't have time to pursue. Any thoughts @ianwremmal?